services/enterprise-gateway/CustomKernelManager/CustomKernelManager.py

```diff
@@ -6,6 +7,9 @@ class CustomKernelManager(RemoteMappingKernelManager):
     async def start_kernel(self, **kwargs: Union[dict[str, Any], None]):
         env = kwargs.get("env", {})
         username = env.get("KERNEL_USERNAME", "unknown")
+        env["TREX__ENDPOINT_URL"] = os.getenv("TREX__ENDPOINT_URL", "")
```

Comment:
since TREX__ENDPOINT_URL is static, we can inject this value here instead from the frontend